core/kazoo_speech/src/kazoo_tts.erl

```diff
@@ -45,7 +47,17 @@ create(<<"flite">>, _Text, _Voice, _Format, _Options) ->
     {'error', 'tts_provider_failure', <<"flite is not available to create TTS media">>};
 create(Provider, Text, Voice, Format, Options) ->
     lager:debug("using provider ~s to create tts", [Provider]),
-    (provider_module(Provider)):create(Text, Voice, Format, Options).
+    Module = provider_module(Provider),
+    Module:create(Text, Voice, Format, Options).
+
+-spec decode(kz_term:ne_binary(), kz_term:ne_binary(), kz_json:object(), any()) -> decode_resp().
```

Comment:
`ProviderData` seems to be `kz_term:api_object()` and not `any()`. Is there a case where it might be something else?